content/en/post/2019-5-15-introducing-oak-ct-log.md

```diff
@@ -5,6 +5,7 @@ excerpt: "Today we are announcing a new Certificate Transparency log called Oak.
 title: "Introducing Oak, a Free and Open Certificate Transparency Log"
 slug: introducing-oak-ct-log
 ---
+Update: Feb. 5 2020, The Let’s Encrypt CT logs are now included in approved log lists and are usable by all publicly-trusted certificate authorities. 
```

Comment:
I am not sure how to note this in the PR, but could this update look like previous ones (https://letsencrypt.org/2016/06/23/defending-our-brand.html)?

Yes, this should do it:

```suggestion
> **Update: Feb. 5 2020**
>
> The Let’s Encrypt CT logs are now included in approved log lists and are usable by all publicly-trusted certificate authorities. 
```

Reply:
(you should be able to directly commit this suggestion with the button above, and see the result there: https://deploy-preview-904--letsencrypt.netlify.com/2019/05/15/introducing-oak-ct-log.html)